pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -0,0 +1,260 @@
+		if deploy.ObjectMeta.CreationTimestamp.IsZero() {
+			deploy.Spec.Selector = &metav1.LabelSelector{
+				MatchLabels: selectorLabels,
+			}
+		}
+
+		deploy.ObjectMeta.Labels = deploymentLabels
+		cephv1.GetCephExporterLabels(cephCluster.Spec.Labels).ApplyToObjectMeta(&deploy.ObjectMeta)
+		k8sutil.AddRookVersionLabelToDeployment(deploy)
+		if cephVersion != nil {
+			controller.AddCephVersionLabelToDeployment(*cephVersion, deploy)
+		}
+		deploy.Spec.Template = corev1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: deploymentLabels,
+			},
+			Spec: corev1.PodSpec{
+				NodeSelector: nodeSelector,
+				InitContainers: []corev1.Container{
+					getCephExporterChownInitContainer(cephCluster),
+				},
+				Containers: []corev1.Container{
+					getCephExporterDaemonContainer(cephCluster, *cephVersion),
+				},
+				Tolerations:       tolerations,
+				RestartPolicy:     corev1.RestartPolicyAlways,
+				HostNetwork:       cephCluster.Spec.Network.IsHost(),
+				Volumes:           volumes,
+				PriorityClassName: cephv1.GetCephExporterPriorityClassName(cephCluster.Spec.PriorityClassNames),
+			},
+		}
+		cephv1.GetCephExporterAnnotations(cephCluster.Spec.Annotations).ApplyToObjectMeta(&deploy.Spec.Template.ObjectMeta)
+		applyPrometheusAnnotations(cephCluster, &deploy.Spec.Template.ObjectMeta)
+
+		return nil
+	}
+
+	return controllerutil.CreateOrUpdate(r.opManagerContext, r.client, deploy, mutateFunc)
+}
+
+func getCephExporterChownInitContainer(cephCluster cephv1.CephCluster) corev1.Container {
+	dataPathMap := config.NewDatalessDaemonDataPathMap(cephCluster.GetNamespace(), cephCluster.Spec.DataDirHostPath)
+	configDir := path.Join(cephCluster.Spec.DataDirHostPath, cephCluster.Namespace)
+	mounts := append(controller.DaemonVolumeMounts(dataPathMap, "", cephCluster.Spec.DataDirHostPath), v1.VolumeMount{Name: "ceph-conf-dir", MountPath: configDir})
+
+	return controller.ChownCephDataDirsInitContainer(
+		*dataPathMap,
+		cephCluster.Spec.CephVersion.Image,
+		controller.GetContainerImagePullPolicy(cephCluster.Spec.CephVersion.ImagePullPolicy),
+		mounts,
+		cephv1.GetCephExporterResources(cephCluster.Spec.Resources),
+		controller.PodSecurityContext(),
+		"",
+	)
+}
+
+func getCephExporterDaemonContainer(cephCluster cephv1.CephCluster, cephVersion cephver.CephVersion) corev1.Container {
+	cephImage := cephCluster.Spec.CephVersion.Image
+	dataPathMap := config.NewDatalessDaemonDataPathMap(cephCluster.GetNamespace(), cephCluster.Spec.DataDirHostPath)
+	configDir := path.Join(cephCluster.Spec.DataDirHostPath, cephCluster.Namespace)
+	cephConfFile := path.Join(configDir, fmt.Sprintf("%s.config", cephCluster.Namespace))
+	volumeMounts := append(controller.DaemonVolumeMounts(dataPathMap, "", cephCluster.Spec.DataDirHostPath), v1.VolumeMount{Name: "ceph-conf-dir", MountPath: configDir})
+
+	container := corev1.Container{
+		Name:            "ceph-exporter",
+		Command:         []string{"/usr/bin/ceph-exporter"},
```

Comment:
Try without the full path, it should be in the path.
```suggestion
		Command:         []string{"ceph-exporter"},
```